component/service/src/main/java/org/exoplatform/social/rest/impl/space/SpaceRestResourcesV1.java

```diff
@@ -232,10 +232,13 @@ public Response createSpace(@Context UriInfo uriInfo,
                                                 "<br />\"visibility\": \"private\"," +
                                                 "<br />\"subscription\": \"validation\"<br />}" 
                                                 , required = true) SpaceEntity model) throws Exception {
-    if (model == null || model.getDisplayName() == null || model.getDisplayName().length() == 0 || model.getDisplayName().length() > 200 || !SpaceUtils.isValidSpaceName(model.getDisplayName())) {
+    if (model == null || model.getDisplayName() == null || model.getDisplayName().length() == 0 || !SpaceUtils.isValidSpaceName(model.getDisplayName())) {
       throw new WebApplicationException(Response.Status.BAD_REQUEST);
```

Comment:
Why don't you change the exception here instead of adding another if block. IMO this is the block to validate the space display name.

Reply:
But this is a generic exception includes many cases, in fact i need a specific exception to inform the user that it is over 200 characters in the space name. WDYT ?

Reply:
Done !